frontend/public/components/RBAC/_rbac.scss

```diff
@@ -3,6 +3,7 @@
 }
 
 .rbac-rule {
+  width: 100%; // since it's parent .co-resource-list__item is a flex container
```

Comment:
nit: should be "its" with no apostrophe :)

Reply:
Rather than a comment, can we encode that?  

```CSS
.co-resource-list__item {
   .rbac-rule {
      width: 100% 
   }
}
```
BEM doesn't approve of the nesting, but the comment reveals these are coupled via nesting anyway.  Perhaps both are helpful?